### DISCUSSION TITLE
Use ImageMagick in AWS Lamba

### DISCUSSION THREAD

**Discussion by mko22** [ImageMagick/ImageMagick#5260] [Help]
I need to use ImageMagick in AWS Lambda. I could zip it and create a layer with it but the problem is that it's installed with apt and not pip so I'm having difficulties with it and I'm stuck with that problem. Here is the issue I get when I use write `from wand.image import Image as Im` and here is the error message

```
Response
{
  "errorMessage": "Unable to import module 'lambda_function': MagickWand shared library not found.\nYou probably had not installed ImageMagick library.\nTry to install:\n  https://docs.wand-py.org/en/latest/guide/install.html",
  "errorType": "Runtime.ImportModuleError",
  "stackTrace": []
}

```

**Comment by gabriel-fsa** [comment]
Hey, did you manage to solve this problem? I have the same 😭

**Comment by 3rdxw** [comment]
> Hey, did you manage to solve this problem? I have the same 😭

Did you fix it?

**Reply by gabriel-fsa** [reply]
Hi, pdfplumber currently removes the dependency on ImageMagick, so probably if you update to the latest version it will work.
